### PR TITLE
Edson Hutton W.I.P.

### PR DESCRIPTION
**Pull request by SUPATURBOLOCITY**

### CODE REVIEW

File: index1.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+ <!-- Nav section - Top Header Nav -->
+   <!-- Top Header Nav -->
+-    <div class="row">	+      
+    <div class="container my-3">
+    <div class="col-sm-12">        
+    <div class="row flex-column">
+    <div class="col-sm-12">
+    <h1 class="text-center">SpaceWalkers</h1>
+             </div>
+             <div class="col-sm-12 align-self-center">
```
Comment (StevieIsmagic) [inline index1.html]: consider consistent indentation

**Review by StevieIsmagic — APPROVED** [review]
Reviewed code live 1on1

**Review by StevieIsmagic — APPROVED** [review]
Reviewed during 1on1 call.

**Comment by dresean** [comment]
- Answers
- well done!
- wish you could be more thorough here and used markdown to format the answers. 
- the utility class for displaying flex is `d-flex`.

- Less / CSS
- beautifully done! the code style has definietely improved this week! 
- loving how you make sure each type of variable/property had their own space.

-Index1
- beautiful execution.

- Notes
- make sure to clean up anything unused, ex. files, code snippets, variables, etc. 
- would have liked to see you complete each part of the assignment
- work on time management

- Overall 
- Amazing job on this one Edson! Glad to your code fu being whipped up into shape! Keep it up!

**Pull request closed without merging**